### DISCUSSION TITLE
OpenShot cask installs latest official build, which is broken in MacOS. Nightly works. What to do?

### DISCUSSION THREAD

**Discussion by hmijail** [Homebrew/discussions#2793] [Casks]
The `openshot-video-editor` cask installs build v2.6.1, which is the latest official one. But the installed app is broken in (at least) MacOS 12.0 and 12.1. This is reported in multiple issues in the OpenShot repo.
Since mid-Dec 2021 there is a nightly build that works. 
My understanding then is that there's 2 problems:
- The mentioned cask installs a non-working app (for MacOS). Should it be removed?
- The working alternative (for MacOS) is not an official release. Should a cask be created in homebrew-cask-versions?

**Comment by miccal** [comment]
https://github.com/Homebrew/homebrew-cask/pull/117496
